src/modules/flow/iio/nodes.c

```diff
@@ -579,6 +607,19 @@ temperature_open(struct sol_flow_node *node, void *data, const struct sol_flow_n
     mdata->iio_base.data_type = DOUBLE;
     mdata->iio_base.config.buffer_size = opts->buffer_size;
     mdata->iio_base.config.sampling_frequency = opts->sampling_frequency;
+    ret = snprintf(mdata->iio_base.config.sampling_frequency_name,
+        sizeof(mdata->iio_base.config.sampling_frequency_name), "%s", "in_temp_");
+    SOL_INT_CHECK_GOTO(ret, >= (int)sizeof(mdata->iio_base.config.sampling_frequency_name), err);
+    SOL_INT_CHECK_GOTO(ret, < 0, err);
```

Comment:
Not a big deal, but we have a macro called `SOL_EXP_CHECK` that an be used here to avoid two checks.